src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -50,14 +52,18 @@
  */
 public class GameRunner {
 
+  public enum GameMode { SWING_CLIENT, HEADLESS_BOT }
+
+  public static final String TRIPLEA_HEADLESS = "triplea.headless";
```

Comment:
All props from the headless server were moved to GameRunner, consolidating to GameRunner so it has teh responsibility of launching games.